### PR TITLE
Refactor ECS instance document

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
- According to the go programming specifications, the maximum length of each line of code should not exceed 120.
The sentence pattern with less than 120 characters is divided into lines, which is easy to read.
- The parameter prefix description uses Specifies as Argument Reference.
- Delete meaningless text description.
- Hyperlink uses English document as destination.
- Use whitespace to instead tab.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
```
NONE
```

**Special notes for your reviewer**:

**Release note**:

```release-note
1. replace prefix description as 'Specifies'.
2. delete meaningless text description of flavor_id, flavor_name, system_disk_size and delete_disks_on_termination.
3. availability_zone hyperlink uses English document as destination.
4. use whitespace to instead tab.
5. a maximum of 120 characters are allowed in one line.
```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: docs/resources/compute_instance.md [hunk 1]
```diff
@@ -185,89 +185,99 @@ function, or the `template_cloudinit_config` resource.
...
-* `system_disk_size` - (Optional, String) The system disk size in GB, The value range is 1 to 1024. Changing this parameter will update the disk. 
-    You can extend the disk by setting this parameter to a new value, which must be between current size and the max size(1024). 
+* `system_disk_size` - (Optional, String) Specifies the system disk size in GB, The value range is 1 to 1024.
     Shrinking the disk is not supported.
 
-* `data_disks` - (Optional, String) An array of one or more data disks to attach to the
-    instance. The data_disks object structure is documented below. Changing this
-    creates a new server.
+* `data_disks` - (Optional, String, ForceNew) Specifies an array of one or more data disks to attach to the instance.
+    The data_disks object structure is documented below.
```
Comment (ShiChangkuo) [inline docs/resources/compute_instance.md:248]: we cannot find the object structure in below, please add it on.

**Comment by ShiChangkuo** [comment]
@Lance52259 `SATA` is not supported, please update the usages, thanks!

File: docs/resources/compute_instance.md (hunk 1 above)
Comment (Lance52259) [reply docs/resources/compute_instance.md:248]: Updated at line 298 to 307.

**Review by Lance52259 — COMMENTED** [review]
> @Lance52259 `SATA` is not supported, please update the usages, thanks!

Already updated data_disks type as 'SSD' to instead of 'SATA'.

**Pull request merged**